src/main/java/io/rebloom/client/cf/IteratorDataPair.java

```diff
@@ -0,0 +1,19 @@
+package io.rebloom.client.cf;
+
+public class IteratorDataPair {
+  private long iteratorValue;
+  private byte[] data;
+
+  public IteratorDataPair(Long iteratorValue, byte[] data) {
```

Comment:
**Please make sure that `iter` returned by `CF.SCANDUMP` fits a signed 64-bit integer. If it's an unsigned 64-bit integer, use String instead of long.**
```suggestion
  public IteratorDataPair(long iteratorValue, byte[] data) {
```

Reply:
I think it makes sense to just use String then, since this value is not really meant to be manipulated by the users, just temporarily stored.

Reply:
Or should we just use `byte[]` for the Iter and forgo any encoding/decoding on the client?

Reply:
@gkorland ping

Reply:
@sazzad16 Turns out that  "In Java SE 8 and later, you can use the long data type to represent an unsigned 64-bit long" https://docs.oracle.com/javase/tutorial/java/nutsandbolts/datatypes.html so I'll stick with long for this then.

Reply:
@bsbodden Just be prepared and careful https://stackoverflow.com/questions/25556017/how-to-use-the-unsigned-integer-in-java-8-and-java-9